src/equations/compound-interest.js

```diff
@@ -18,8 +43,8 @@ export const compoundInterestWithMonthlyContributions =
   numberOfYears
   ) => {
     const totalContrubutions = monthlyContributions * numberOfYears * 12;
-    const principalAfterInterest = principal * Math.pow(1+((annualInterestRate/100)/compoundsPerYear),compoundsPerYear*numberOfYears);
-    const contributationsAfterInterest = monthlyContributions*((Math.pow((1+((annualInterestRate/100)/compoundsPerYear)),(compoundsPerYear*numberOfYears))-1)/((annualInterestRate/100)/compoundsPerYear));
+    const principalAfterInterest = principal * pow(1+((annualInterestRate/100)/compoundsPerYear),compoundsPerYear*numberOfYears);
```

Comment:
we should create unit tests for this file before merging

Reply:
100%